lib/lackeys/registry.rb

```diff
@@ -121,8 +126,10 @@ def self.default_object_hash
                          :load_callbacks,
                          :default_object_hash
 
-    def method?(method_name)
-      value_hash.keys.include? method_name.to_sym
+    def method?(method_name, return_origin = false)
+      res = value_hash[method_name.to_sym]
+      return !res.nil? unless return_origin
+      res.nil? ? nil : (res[:observers].size == 1 ? res[:observers].first : res[:observers])
```

Comment:
Might be a better contract with the Registry for the results to always be an Array. Just so anyone using this can assume that they would always get consistent data.